Tasks/AzureRmWebAppDeploymentV4/operations/TaskParameters.ts

```diff
@@ -98,6 +98,19 @@ export class TaskParametersUtility {
             taskParameters.AdditionalArguments = '-retryAttempts:6 -retryInterval:10000';
         }
 
+        if(taskParameters.isLinuxApp && taskParameters.ScriptType) {
```

Comment:
Can you please make this is one method like ScriptypeLinuxParameter parsing to make it modular ?

Reply:
moved to new method.